lib/modules/manager/conan/artifacts.ts

```diff
@@ -0,0 +1,100 @@
+import { quote } from 'shlex';
+import { TEMPORARY_ERROR } from '../../../constants/error-messages';
+import { logger } from '../../../logger';
+import { exec } from '../../../util/exec';
+import type { ExecOptions } from '../../../util/exec/types';
+import {
+  findLocalSiblingOrParent,
+  readLocalFile,
+  writeLocalFile,
+} from '../../../util/fs';
+import { getGitEnvironmentVariables } from '../../../util/git/auth';
+import type { UpdateArtifact, UpdateArtifactsResult } from '../types';
+
+async function conanUpdate(conanFilePath: string): Promise<void> {
+  const command = `conan lock create ${quote(conanFilePath)} --lockfile=""`;
+
+  const execOptions: ExecOptions = {
+    extraEnv: { ...getGitEnvironmentVariables(['conan']) },
+    docker: {},
+  };
+
+  await exec(command, execOptions);
+}
+
+export async function updateArtifacts(
+  updateArtifact: UpdateArtifact,
+): Promise<UpdateArtifactsResult[] | null> {
+  const { packageFileName, updatedDeps, newPackageFileContent, config } =
+    updateArtifact;
+
+  logger.debug(`conan.updateArtifacts(${packageFileName})`);
+
+  const isLockFileMaintenance =
+    config.updateType === 'lockFileMaintenance' ||
+    config.isLockFileMaintenance === true;
```

Comment:
Shouldn't you delete the existing file, or run some alternative command, if you are doing lock file maintenance? Remember it's meant to be a full refresh of the lock file